sky/clouds/azure.py

```diff
@@ -136,35 +136,40 @@ def get_default_instance_type(
     def _get_image_config(self, gen_version, instance_type):
         # az vm image list \
         #  --publisher microsoft-dsvm --all --output table
-        # nvidia-driver: 495.29.05, cuda: 11.5
-
-        # The latest image 2022.09.14/2022.08.11/22.06.10/22.05.11/
-        # 22.04.27/22.04.05 has even older nvidia driver 470.57.02,
-        # cuda: 11.4
+        # nvidia-driver: 535.54.03, cuda: 12.2
+        # see: https://github.com/Azure/azhpc-images/releases
```

Comment:
```suggestion
        # see: https://github.com/Azure/azhpc-images/releases/tag/ubuntu-hpc-20230803
```
let's link to specific tag. otherwise it will change in the future